hub/core/transform/transform.py

```diff
@@ -106,6 +106,11 @@ def eval(
         ds_out.storage.autoflush = False
 
         tensors = list(ds_out.tensors)
+
+        # Initialize chunk id encoders
+        for tensor in tensors:
+            ds_out[tensor].chunk_engine.chunk_id_encoder
```

Comment:
does this do anything?

Reply:
Yes. It initializes the chunk id encoders. Previously `len(tensor)` would initialize the chunk id encoder if it didnt exist, but I changed that behavior.